doc/user_guide/user_guide.md

```diff
@@ -154,15 +170,50 @@ If you want to provide the name of the root bucket on the command line then plea
 
 ### Bucket Operations
 
+#### Listing Contents Of a Bucket Or a Directory In a Bucket
+
+With `bfsc ls` you can list the contents of a directory in the bucket. If you do not specify a directory then the `ls` command will list all files and directories in the bucket on top level.
+
+| Command            | Output                                  |
+|--------------------|-----------------------------------------|
+| `bfsc ls`          | Contents of the bucket's root directory |
+| `bfsc ls folder/a` | Contents of directory `folder/a`        |
+
+Example output
+```
+a.txt
+b.txt
+folder\
+```
+
+BFSC highlights folders with a trailing slash `/` to separate them from files potentially having the same name.
+
+If you want to list all root buckets rather than the contents of a specific bucket, then please ensure to not not set `bucket` in your profile.
+
+Example output for listing root buckets
+```
+default\
+simba\
+```
+
+With option `-r` or `--recursive` you can obtain a recursive listing of all contents inside a directory of a bucket. However when listing root *buckets* then BFSC will ignore option `-r`.
+
+```shell
+$ bfsc ls -r folder
```

Comment:
```suggestion
$ bfsc ls -r bfs:/folder
```
❓

Reply:
As bfsc is not designed to list files in the local filesystem, anyway, I left the protocol prefix intentionally.
But I will add a dedicated test to ensure this works as expected, though.